index.html

```diff
@@ -7658,8 +7658,34 @@ <h5>Presentational Role Inheritance</h5>
 				    <li>If <code>aria-valuemin</code> is missing or not a <a href="#valuetype_number">number</a>, it defaults to 0 (zero). </li>
 				    <li>If <code>aria-valuemax</code> is missing or not a <a href="#valuetype_number">number</a>, it defaults to 100. </li>
 				</ul>
-				<p>Authors MUST set the <pref>aria-valuenow</pref> attribute. If aria-valuenow is missing or has an unexpected value, browsers MAY implement the repair techniques specified in the <a href="#authorErrorDefaultValuesTable">section describing handling author errors in states and properties</a>, which are equivalent to the repair techniques for <code>&lt;input type="[^input/type/range^]"&gt;</code> in HTML.</p>
+				<p>Authors MUST set either the <pref>aria-valuenow</pref> attribute, the <pref>aria-valuetext</pref> attribute, or both. If both the <code>aria-valuenow</code> and <code>aria-valuetext</code> attributes are missing or <code>aria-valuenow</code> has an unexpected value, <a>user agents</a> MAY implement the repair techniques specified in the <a href="#authorErrorDefaultValuesTable">section describing handling author errors in states and properties</a>, which are equivalent to the repair techniques for <code>&lt;input type="[^input/type/range^]"&gt;</code> in HTML. However, both <a>user agents</a> and <a>assistive technologies</a> SHOULD NOT expose a fallback value for <code>aria-valuenow</code> if the author has only specified the <code>aria-valuetext</code> attribute.</p>
 				<p>Elements with the role <code>slider</code> have an implicit <pref>aria-orientation</pref> value of <code>horizontal</code>.</p>
+				<div class="note">
+					<p>It is usually advisable for a slider to express its current value as a parsable <a href="#valuetype_number">number</a> using <pref>aria-valuenow</pref>. However, there are use cases where <pref>aria-valuetext</pref> alone provides a superior user experience.</p>
+					<p>For example, pain can be measured along a range of values in which each value is most helpfully expressed by descriptive text rather than an unintuitive number.</p>
+				</div>
+				<pre class="example highlight">
+&lt;!-- Warning: this code example is incomplete. --&gt;
+&lt;div id="label"&gt;Pain rating&lt;/div&gt;
+&lt;div id="pain" role="slider" tabindex="0" aria-labelledby="label" aria-valuetext="No pain"&gt;
+  &lt;ol id="ratings" aria-hidden="true"&gt;
+    &lt;li&gt;No pain&lt;/li&gt;
+    &lt;li&gt;Hardly noticeable pain&lt;/li&gt;
+    &lt;li&gt;Somewhat distracting pain&lt;/li&gt;
+    &lt;li&gt;Pain interrupts some activities&lt;/li&gt;
+    &lt;li&gt;Pain prevents daily activities&lt;/li&gt;
+    &lt;li&gt;Unbearable pain, cannot do anything&lt;/li&gt;
+    &lt;li&gt;As bad as pain can be, nothing else matters&lt;/li&gt;
+  &lt;/ol&gt;
+&lt;/div&gt;
+&lt;script&gt;
+  document.querySelectorAll('#ratings li').forEach(function(rating, index) {
```

Comment:
Even with the comment about the example being incomplete, I think I'd lean towards leaving off the script entirely. Maybe adding a comment above the `<ol>` clarifying that those are the visible marks & labels along the slider.

The reason being that even with all the warnings in the world, people still go to the ARIA spec and say "oh this is how I should do things", then go tell other people "this is what we should be doing, this is the spec" 😅. And this script example doesn't include keyboard, drag, etc., and with `aria-hidden` + click events, mobile SRs and voice control won't be able to interact, etc. etc. (that's not a critique of your example code, I know showing all that is impossible in a small example, but just a reason to not demo any of the script since anything we do will be super inaccessible in an aria slider scenario)

Reply:
Oof, yes. 🤦🏻  I agonized a bit over how far to take this.

In support of this spec change, I was hoping that the code example would at least illustrate the essential aspect of _only_ `aria-valuetext` being updated and `aria-valuenow` being absent altogether. I’ll take another run at this.